lib/rack/test.rb

```diff
@@ -1,3 +1,5 @@
+# frozen-string-literal: true
```

Comment:
Isn't it supposed to be `frozen_string_literal` or is this the same thing/compatible with dashes?

Reply:
Both work:

```
$ printf "# frozen-string-literal: true\np ''.frozen?" | ruby
true
$ printf "# frozen_string_literal: true\np ''.frozen?" | ruby 
true
```

Reply:
I've only ever seen `frozen_string_literal` used in practice, is `frozen-string-literal` common?

Reply:
I think we should go with the more common style unless there is a good reason not to.

Reply:
OK, I pushed another commit for that.